packages/valory/connections/p2p_libp2p/tests/test_aea_cli.py

```diff
@@ -89,9 +89,6 @@ def run_aea_cli_test(self) -> None:
         missing_strings = self.missing_from_output(process, check_strings)
         assert not missing_strings
 
-        self.terminate_agents(process)
-        assert self.is_successfully_terminated(process)
-
```

Comment:
please comment

Reply:
Covered in teardown_class I think

Reply:
exactly, and better because this resulted in flaky behaviour on windows